sycl/plugins/level_zero/CMakeLists.txt

```
@@ -16,15 +16,12 @@ if (NOT DEFINED LEVEL_ZERO_LIBRARY OR NOT DEFINED LEVEL_ZERO_INCLUDE_DIR)
     GIT_TAG           ${LEVEL_ZERO_LOADER_TAG}
   )
 
-  # sccache is not compatible with /Zi flag, disable sccache
-  set(CMAKE_CXX_COMPILER_LAUNCHER_BAK "${CMAKE_CXX_COMPILER_LAUNCHER}")
-  set(CMAKE_CXX_COMPILER_LAUNCHER "")
-
   # Workaround warnings/errors for Level Zero build
   set(CMAKE_CXX_FLAGS_BAK "${CMAKE_CXX_FLAGS}")
   if (WIN32)
-    #FIXME: Level Zero build fails with /DUNICODE
-    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /UUNICODE")
+    # FIXME: Level Zero build fails with /DUNICODE
+    # USE_Z7 forces use of /Z7 instead of /Zi which is broken with sccache
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /UUNICODE -DUSE_Z7")
```

```suggestion
    # FIXME: Level Zero build fails with /DUNICODE
    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /UUNICODE")
    # USE_Z7 forces use of /Z7 instead of /Zi which is broken with sccache
    set(USE_Z7 ON)
```
USE_Z7 is cmake var it seems, not C macro def